tests/access_test.go

```diff
@@ -53,6 +54,51 @@ var _ = Describe("[rfe_id:500][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		tests.BeforeTestCleanup()
 	})
 
+	Describe("With the kubevirt-controller service account", func() {
+		table.DescribeTable("should verify permissions on resources are correct for kubevirt-controller", func(resource string) {
+
+			controllerVerbs := make(map[string]string)
+			controllerVerbs["get"] = "yes"
```

Comment:
Maybe to simplify the test, you could initialize this map with the same value for all the verbs and then adjust in your switch case? Or maybe even pass a struct in the describe table?

Reply:
I was following the previously existing code on this set of tests. 

There are 8x6 = 48 individual compares: 8 verbs for 6 subjects. This block may extend in the future to cover other subjects, e.g. those in `cdi.kubevirt.io` or `snapshot.kubevirt.io`. 

In our added block here, the logic was:
- Since there are several subjects take the same set of verbs, we first assign expected value to the 8 verbs to reflect the most common set.
- Then we use `switch` to overwrite a small amount of cases (4 cases here). 

I have not got the full convention and the preferred priority of code patterns on Ginkgo test writing, Are you suggesting this pattern? ( I can see it's clearer, with tiny redundant ). Is the below what you suggesting? I'd love to switch to it. Thanks.

```
// Pseudo
controllerVerbs["list"] = "yes"
controllerVerbs["watch"] = "yes"
controllerVerbs["update"] = "no"
controllerVerbs["delete"] = "no"
controllerVerbs["deletecollection"] = "no"

switch {
case vm, vmi, vmireplicateset: 
    ...["get"] = "yes"
    ...["patch"] = "yes"
    ...["create"] = "no"
case vmmigrate: 
    ...["get"] = "yes"
    ...["patch"] = "yes"
    ...["create"] = "yes"
case vmipreset: 
    ...["get"] = "no"
    ...["patch"]  = "no"
    ...["create"] = "no"
case kubevirt: 
    ...["get"] = "yes"
    ...["patch"] = "no"
    ...["create"] = "no"
}           
```